torch/quantization/fx/_equalize.py

```diff
@@ -92,6 +97,8 @@ def calculate_qparams(self):
 
         return scale_input, zero_point_input
 
+    with_args = classmethod(_with_args)
```

Comment:
should input observer and weight observer inherit from ObserverBase? https://github.com/pytorch/pytorch/blob/master/torch/quantization/observer.py#L47

Reply:
Sure! I can change that.